src/index.scss

```diff
@@ -30,15 +30,17 @@ input {
   border: solid 0.5px $grey-light;
   border-radius: 4px;
   padding: 10px;
-  font-size: 1.25rem;
+  font-size: 1.4rem;
+  font-family: "Montserrat";
```

Comment:
With font-family its good to have backup fonts incase this one is not able to load on a persons computer. Here's a good short article on it - https://www.phpied.com/psa-always-add-a-generic-font-family-backup/